gateway/src/apicast/policy/custom_metrics/Readme.md

```diff
@@ -0,0 +1,73 @@
+# Custom metrics policies
+
+This policy adds the availability to add metrics after the Upstream API
```

Comment:
I'd add somewhere that every metric used in this policy needs to be created manually in the admin portal.
Some users might expect APIcast to automatically create them when needed.